### PR TITLE
Remove panel underline of button

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

**Review by tupui — COMMENTED** [review]
Actually we have to change the css variable instead. (and also I tested this locally, we should add a margin 15px to `.panel-underline`)

**Comment by jarrodmillman** [comment]
@tupui This should work.

**Comment by tupui** [comment]
As discussed, I will take over this PR.

File: assets/theme-css/panel.css [hunk 1]
```diff
@@ -30,8 +30,9 @@
     height: 140px;
     min-width: 275px;
     width: 335px;
-    margin: 25px auto;
+    margin: 25px 15px 25px 15px;
     border-radius: 3px;
+    box-shadow: 0px 8px 15px var(--colorShadow);
```
Comment (jarrodmillman) [inline assets/theme-css/panel.css:35]: I added it when I removed the panel-underline thing.  Now that it is back, I don't think this is needed.

**Comment by jarrodmillman** [comment]
I tested this on both learn and blog and it LGTM!  I made one comment regarding removing one line that I don't think is necessary.

File: assets/theme-css/panel.css (hunk 1 above)
Comment (tupui) [inline assets/theme-css/panel.css:35]:
```suggestion
```

Comment (tupui) [reply assets/theme-css/panel.css]: Done

**Pull request merged**